client/src/utilities/externalopen.h

```diff
@@ -0,0 +1,33 @@
+/*
+  This file is part of FatCRM, a desktop application for SugarCRM written by KDAB.
+
+  Copyright (C) 2019 Klarälvdalens Datakonsult AB, a KDAB Group company, info@kdab.com
+  Authors: Kevin Funk <kevin.funk@kdab.com>
+
+  This program is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 2 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef EXTERNALOPEN_H
+#define EXTERNALOPEN_H
+
+class QString;
+class QWidget;
+
+class ExternalOpen
```

Comment:
a namespace would be enough (no need to generate default ctor/dtor, and allow writing "new ExternalOpen"...)